packages/core/src/features/cohort/availableCohortsSlice.ts

```diff
@@ -548,26 +564,28 @@ interface CopyCohortParams {
  *
  *
  * The slice exports the following actions:
- * setCohortList() - set saved cohort to the adapter that comes from the server
- * addNewCohort() - create a new cohort
- * addNewCohortWithFilterAndMessage - create a cohort with the passed filters and message id
- * copyCohort - create a copy of the cohort with sourceId to a new cohort with destId
- * updateCohortName(name:string): changes the current cohort's name
- * updateCohortFilter(filters: FilterSet): update the filters for this cohort
- * removeCohortFilter(filter:string): removes the filter from the cohort
- * clearCohortFilters(): removes all the filters by setting them to the default all GDC state
- * setCurrentCohortId(id:string): set the id of the current cohort, used to switch between cohorts
- * clearCaseSet(): resets the caseSet member to all GDC
- * removeCohort(): removes the current cohort
- * setCohortMessage(): sets the current cohort message
- * clearCohortMessage(): clears the current message by setting it to undefined
- * addNewCohortGroups(): adds groups of filters to the current cohort
- * removeCohortGroup(): removes a group of filters from the current cohort
+ * * setCohortList() - set saved cohort to the adapter that comes from the server
```

Comment:
just to be sure if double `* *` is needed here.

Reply:
It formats the data for IDEs to show it as a bulleted list in some IDE, so more for, than the doc. I changed them to `-`